backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -3014,7 +3014,7 @@ void writeVideo(VmDevice device) {
         if (mdevDisplayOn) {
             writer.writeAttributeString("type", "none");
         } else {
-            writer.writeAttributeString("type", device.getDevice());
+            writer.writeAttributeString("type", getVideoType(device.getDevice()));
```

Comment:
do you think we should also have the driver name element? to run older OSes? https://libvirt.org/formatdomain.html#video-devices , https://libvirt.org/formatdomain.html#virtio-related-options

Reply:
> do you think we should also have the driver name element? to run older OSes?

Can you please elaborate why you think it is needed? I don't understand the problem.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1784049 will describe it.
There is already `legacyVirtio` in libvirtxmlbuilder to use.

Reply:
`virtio-transitional` driver is absent for video devices, so I use `vga` as fallback, if `legacyVirtio` is true.